.github/workflows/main-build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build-push-container:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Technically, this version doesn't need to be changed. I don't mind upgrading if CI will pass.